src/status_im/common/wizard.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im.common.wizard
+  (:require [utils.re-frame :as rf]))
+
+(defn- wizard-find-next-screen
+  [db flow-config current-screen]
+  (->> flow-config
+       (filter (fn [{:keys [skip-step? screen-id]}]
+                 (and (not= screen-id current-screen)
+                      (not (and (fn? skip-step?) (skip-step? db))))))
+       first))
+
+(rf/reg-event-fx
+ :navigation/wizard-forward
```

Comment:
given that this is no longer in navigation we should probably update the event name accordingly.
e.g `:common.wizard/navigate-forward` or something like. `:navigation/` should really be reserved for events in that core navigation ns.

Reply:
Also it was asked that we move the wizard stuff into wallet for the time being. Perhaps we move this to `status-im/wallet/common/wizard` ?

If it's needed beyond the wallet we can revisit this 👍 

This will also tie in nicely with your solution @Parveshdhull to have one event containing the various configs.

Reply:
Done